src/components/CreationForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useContext } from 'react'
+import { Link } from 'react-router-dom'
+import { GameContext } from '../store/GameContext'
+
+const initialState = {
+	code: '',
+	talent: '',
+	terms: '',
+	count: '',
+	intervals: ''
+}
+
+export const CreationForm = () => {
+	const { dispatch, state } = useContext(GameContext)
+
+	const handleChange = (event) => {
+		const { value, id } = event.target
+		initialState[id] = value
+		dispatch({ type: 'SET_CREATION_CARD', creationCard: initialState })
+	}
+
+	return (
+		<form className='creation-form'>
+			<label className='choice' htmlFor='talent'>
+				Choose a talent
+			</label>
+			<select className='talents' id='talent' value={state.talent} onChange={handleChange}>
+				<option hidden> Talents </option>
+				<option name='mind'>Mind</option>
+				<option name='spirit'>Spirit</option>
+				<option name='wisdom'>Wisdom</option>
+				<option name='vitality'>Vitality</option>
+			</select>
+
+			<label className='choice' htmlFor='count'>
+				Choose a number between 1 and 13.
+			</label>
+			<input
+				className='slider'
+				id='count'
+				type='range'
+				name='count'
+				min='1'
+				max='13'
+				value={state.count}
+				onChange={handleChange}
+			/>
+
+			<label className='choice' htmlFor='terms'>
+				Write your terms.
+			</label>
+			<input
+				className='terms '
+				id='terms'
+				type='text'
+				name='terms'
+				placeholder='Enter your challenge here'
+				value={state.terms}
+				onChange={handleChange}
+				autoComplete='off'
+			/>
+
+			<label className='choice' htmlFor='intervals'>
+				Choose an interval.
+			</label>
+			<select className='intervals ' id='intervals' value={state.intervals} onChange={handleChange}>
+				<option hidden> Intervals </option>
```

Comment:
same here

Reply:
I'm not so sure how I'll go about refactoring this behavior in CSS. What's bad about it?